addon/components/paper-ink-bar.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
```

Comment:
Use object destructuring: `const { Component } = Ember;`

Let's avoid to use `Ember.` in our code.
Details in the CONTRIBUTING.md file.